scientific/edf.ksy

```diff
@@ -0,0 +1,182 @@
+meta:
+  id: edf
+  file-extension: edf
+  endian: le
+  license: MIT
+
```

Comment:
`doc` and `doc-ref` are mandatory. Also a link to some small samples is desireable.